pytest_plugins/metadata_markers.py

```diff
@@ -191,8 +191,8 @@ def pytest_collection_modifyitems(items, config):
             # Add marker starting at smallest docstring scope
             # only add the mark if it hasn't already been applied at a lower scope
             doc_component = component_regex.findall(docstring)
-            if doc_component and 'component' not in item_mark_names:
-                item.add_marker(pytest.mark.component(doc_component[0]))
+            if doc_component and 'CaseComponent' not in item_mark_names:
```

Comment:
why did you change the `component` to `CaseComponent`. we are adding `component` as marker to the test not `CaseComponent`. Or did I missed anything here ?

Reply:
It searches for CaseComponent docstring in the test.

Reply:
@jyejare I think it should be `component` right ? `CaseComponent` would not be a right marker to check as it it will not be there. As the we are adding the component marker.

Reply:
@shweta83 Here @omkarkhatavkar is right we are adding a `keypair` of `component=<somcomponent>` to the the test item.

In below `compoenent not in item_mark_names` verifies that the component marker is not already added to the item.
```
if doc_component and 'component' not in item_mark_names:
```